packages/gated-content/package.json

```diff
@@ -71,10 +71,16 @@
     "prettier": "^2.8.4",
     "ts-jest": "^29.0.5",
     "typescript": "^4.9.5",
-    "wait-for-expect": "^3.0.2"
+    "wait-for-expect": "^3.0.2",
+    "zod": "^3.20.6"
   },
   "peerDependencies": {
-    "ethers": "^5.7.2"
+    "@ethersproject/contracts": "^5.7.0",
```

Comment:
just for me to get it right, what's the reason we keep part of ethers in `peerDependencies` and part in `dependencies`?

Reply:
The ones in `dependencies` are used directly by us. The ones in `peerDependencies` are used by sub-dependencies. The version number should help install the correct version that goes with the rest of the repo dependencies on the `ethersproject` package ecosystem.